release/Makefile

```
@@ -21,6 +21,8 @@ out/opam-full-$(VERSION).tar.gz:
 	mkdir -p out
 	cd out && curl -OfL $(FULL_ARCHIVE_URL) || { \
 	  git clone $(GIT_URL) -b $(TAG) --depth 1 opam-full-$(VERSION); \
+	  test "`sed -n "s/^let default.*(\([0-9]\),\([0-9]\))/\1.\2/p" src/client/opamCLIVersion.ml`" \
+	    = "`echo $VERSION | cut -f 1-2 -d '.'`" ; \
```

Awesome! 😀 Slightly more hardened:
```suggestion
	  test "`sed -n "/let *default *=/s/.*(\([0-9]*\)[^0-9]*\([0-9]*\).*/\1.\2/p" src/client/opamCLIVersion.ml`" \
	    = "`echo $VERSION | cut -f 1-2 -d '.'`" ; \
```
It's worth a comment in `opamCLIVersion.ml` that the line is analysed (cf. [things like this](https://github.com/ocaml/ocaml/blob/trunk/tools/pre-commit-githook#L16-L18)).

Is this release script used to build the 2.0.x releases as well, or do we use the release script on the 2.0 branch?

---

Good question... @AltGr ?

---

Moved as a ci check, on master push